crates/ruma-identity-service-api/src/invitation/store_invitation/v2.rs

```diff
@@ -0,0 +1,95 @@
+//! [POST /_matrix/identity/v2/store-invite](https://matrix.org/docs/spec/identity_service/r0.3.0#post-matrix-identity-v2-store-invite)
+
+use ruma_api::ruma_api;
+use ruma_common::thirdparty::Medium;
+use ruma_identifiers::{MxcUri, RoomAliasId, RoomId, UserId};
+
+ruma_api! {
+    metadata: {
+        description: "Store pending invitations to a user's 3PID.",
+        method: POST,
+        name: "store_invitation",
+        path: "/_matrix/identity/v2/store-invite",
+        authentication: AccessToken,
+        rate_limited: false,
+    }
+
+    request: {
+        /// The literal string `email`.
+        pub medium: &'a Medium,
+
+        /// The email address of the invited user.
+        pub address: &'a str,
+
+        /// The Matrix room ID to which the user is invited.
+        pub room_id: &'a RoomId,
+
+        /// The Matrix user ID of the inviting user.
+        pub sender: &'a UserId,
+
+        /// The Matrix room alias for the room to which the user is invited. This should be
+        /// retrieved from the `m.room.canonical` state event.
```

Comment:
When an item's docs consist of multiple lines, it should almost always split up into multiple paragraphs. Rustdoc splits after the first paragraph for overview vs. full description of an item. Not sure whether this applies to fields, but even if it doesn't it could in the future.

```suggestion
        /// The Matrix room alias for the room to which the user is invited.
        ///
        /// This should be retrieved from the `m.room.canonical` state event.
```

Reply:
Same below.